CONTRIBUTING.md

```diff
@@ -142,6 +142,14 @@ sure they are not merging changes that do not have the needed updates to documen
 
 Before submitting your change, you should test it. But since this is not setup yet, we'll forgive you.
 
+## Automated CI Testing
+
+We currently have two Jenkins jobs that run automatically when a pull request
+is opened, to build an Aquarium image and run it on openSUSE Tumbleweed and
+openSUSE Leap. If these tests fail and you need to trigger a re-run, add a
+comment to the PR which says `jenkins run tumbleweed` or `jenkins run leap` as
+needed.
```

Comment:
While we currently have 2 jenkins jobs for doing integration testing, we also have github checks that run the unit tests. It might be worth mentioning that given the title of "automated ci testing".

Reply:
I agree it would be useful saying which GitHub Actions respond to what.

Reply:
I've added a mention of GitHub actions and unit tests, but I wasn't sure exactly what to describe, so it's pretty generic.